hack/build-images.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
add error handling for missing required parameters

Reply:
ok, I can add those but this is to be only called from the makefile and `set -u` kind of does that already

Reply:
done